### PR TITLE
UI Improvements

### PR DESCRIPTION
**Pull request by sampoder**
Based on image sent on Slack

### CODE REVIEW

File: issue_maker.py [hunk 1]
```diff
@@ -465,8 +465,9 @@ def latest_version():
...
...//www.w3.org/2000/svg' viewBox='0 0 4 5'%3e%3cpath fill='%23343a40' d='M2 0L0 2h4zm0 5L0 3h4z'/%3e%3c/svg%3e\" ) no-repeat right .75rem center/8px 10px;background-color:#fff;border:1px solid #ced4da;border-radius:.25rem;-webkit-appearance:none;-moz-appearance:none;appearance:none}.custom-select:focus{border-color:#80bdff;outline:0;box-shadow:0 0 0 .2rem rgba(0,123,255,.25)}.custom-select:focus::-ms-value{color:#495057;background-color:#fff}.custom-select[multiple],.custom-select[size]:not([size=\" 1\" ]){height:auto;padding-right:.75rem;background-image:none}.custom-select:disabled{color:#6c757d;background-color:#e9ecef}.custom-select::-ms-expand{display:none}.custom-select-sm{height:calc(1.5em + .5rem + 2px);padding-top:.25rem;padding-bottom:.25rem;padding-left:.5rem;font-size:.875rem}.custom-select-lg{height:calc(1.5em + 1rem + 2px);padding-top:.5rem;padding-bottom:.5rem;padding-left:1rem;font-size:1.25rem}.custom-file{position:relative;display:inline-block;width:100%;height:calc(1.5em + .75rem + 2px);margin-bottom:0}.custom-file-input{position:relative;z-index:2;width:100%;height:calc(1.5em + .75rem + 2px);margin:0;opacity:0}.custom-file-input:focus~.custom-file-label{border-col
```
Comment (aadibajpai) [inline issue_maker.py]: This is not a very maintainable way of making a template in Python. If I later have to edit it even slightly then I will have a super hard time.

**Review by aadibajpai — CHANGES_REQUESTED** [review]
The templates and the code in the `tab.py` should help you figure out how templates are rendered. 
https://flask.palletsprojects.com/en/1.1.x/quickstart/#rendering-templates should be helpful as well 😄

File: issue_maker.py (hunk 1 above)
Comment (aadibajpai) [inline issue_maker.py]: Additionally, the design could be similar to the SwagLyrics application, we don't want to start off with something too complex :)

Comment (sampoder) [reply issue_maker.py]: Ummm.... totally understand I'll redo it to be more like the main SwagLyrics and not in here

Comment (aadibajpai) [reply issue_maker.py]: That would be great. 👍

Comment (sampoder) [reply issue_maker.py]: Sorry if that first bit sounder sarcastic, I just realised it could be seen as rude. I mean ummm in like how did I over see it

Comment (aadibajpai) [reply issue_maker.py]: Oh haha no it was okay. I certainly didn't think it was rude. Don't worry about it.

**Comment by aadibajpai** [comment]
You can use normal html/css/js files for the template. Like https://github.com/SwagLyrics/SwagLyrics-For-Spotify/tree/master/swaglyrics/static and then the html needs just some slight modification to work with python and flask.

**Comment by sampoder** [comment]
I think I haved changed it, let me know if it still needs improvement :)

File: issue_maker.py [hunk 2]
```diff
@@ -464,10 +464,8 @@ def latest_version():
 @limiter.exempt
 def hello():
     with open('unsupported.txt', 'r') as f:
-        data = f.read()
-    data = ('Repositories and stuff at https://github.com/SwagLyrics <br><br>Unsupported Songs <br>--------------------'
-            '----<br><br>' + data).replace('\n', '<br>')
-    return data
+        data = f.read().splitlines()
```
Comment (aadibajpai) [inline issue_maker.py]:
```suggestion
        data = f.readlines()
```
This would do the same thing.

File: templates/hello.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel='shortcut icon' type='image/x-icon' href='../static/favicon.ico'/>
+    <link rel = "stylesheet" type = "text/css" href = "{{url_for('static', filename='style.css')}}">
+    <title>SwagLyrics - Unsupported Songs</title>
```
Comment (aadibajpai) [inline templates/hello.html]:
```suggestion
    <title>SwagLyrics API</title>
```

File: templates/hello.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+</head>
+<body>
+<h2>Unsupported Songs - <a href ="https://github.com/SwagLyrics">Swag Lyrics</a></h2>
```
Comment (aadibajpai) [inline templates/hello.html]:
```suggestion
<h2>Unsupported Songs - <a href ="https://github.com/SwagLyrics">SwagLyrics</a></h2>
```

File: templates/hello.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+<div id="lyrics">
+{{line}}<br>
+    
+    {%for x in unsupported_songs} 
+  
+    {{x}}<br><br>
```
Comment (aadibajpai) [inline templates/hello.html]:
```suggestion
    {{x}}<br>
```
I think an extra line isn't necessary

**Review by aadibajpai — CHANGES_REQUESTED** [review]
Much more promising, just needs these minor tweaks.

File: templates/hello.html [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+{{line}}<br>
+    
+    {%for x in unsupported_songs} 
+  
+    {{x}}<br><br> 
+    
+    {%endfor%} 
+    
+    </div>
+</script>
```
Comment (aadibajpai) [inline templates/hello.html]: where does this script tag start?

Comment (sampoder) [reply templates/hello.html]: hmmmmm.... I wonder maybe nowhere. let me fix it oops

**Comment by sampoder** [comment]
All fixed!

**Comment by aadibajpai** [comment]
> All fixed!

Okay, I'm going to merge. Hopefully nothing breaks 🤞

**Comment by aadibajpai** [comment]
> All fixed!

Okay, I'm going to merge. Hopefully nothing breaks 🤞

**Pull request merged**